amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/ExchangeServiceImpl.java

```diff
@@ -189,4 +189,17 @@ public CompletableFuture<Integer> exchangeBound(NamespaceName namespaceName, Str
         return future;
     }
 
+    @Override
+    public CompletableFuture<Void> queueBind(NamespaceName namespaceName, String exchange, String queue,
+                                             String routingKey, Map<String, Object> arguments) {
+        return exchangeContainer.asyncGetExchange(namespaceName, exchange, false, null)
```

Comment:
If exchange is the system default (amp.topic, amqp.direct... ..) , should be created before bind

Reply:
I move the build-in exchange name check to method `ExchangeContainer#asyncGetExchange`.